common/lib/capa/capa/templates/optioninput.html

```diff
@@ -13,12 +13,13 @@
     </select>
 
   <span id="answer_${id}"></span>  
-  <span class="status ${status.classname}"
-      id="status_${id}"
-      aria-describedby="input_${id}">
-    <span class="sr">${value|h} - ${status.display_name}</span>
-  </span>
-
+  <div class="indicator_container">
+    <span class="status ${status.classname}"
```

Comment:
Are there line breaks here due to a max line length, or for visual formatting? I'd prefer it if these were on one line if possible.